doc/lsst.ap.pipe/pipeline-bps.rst

```diff
@@ -39,45 +31,47 @@ Typically it will contain info about the processing campaign, desired input and
 
 Here's an example .yaml file governing what gets passed to pipetask.
 It is simply a slightly modified version of the example in `ap_pipe/bps/bps_ApPipe.yaml <https://github.com/lsst/ap_pipe/blob/main/bps/bps_ApPipe.yaml>`_.
-Again, as of March 2022 this assumes you are running on one of the lsst-devl machines at NCSA.
+Again, as of September 2023 this assumes you are running on one of the lsst-devl machines at USDF.
 
 .. code-block:: yaml
 
-
   # Path to the pipeline to run
-
   pipelineYaml: '/path/to/my/pipeline.yaml'
-  # Format for job names and job output filenames
-  templateDataId: '{tract}_{patch}_{band}_{visit}_{exposure}_{detector}'
+  
+  # Names to help organize runs
   project: ApPipe
   campaign: my_example
   
   # Directory where files associated with your submission, such as logs, will go.
+  # Default is shown.
   submitPath: ${PWD}/bps/{outputRun}
-  computeSite: ncsapool
+  
+  # Specify WMS plugin (HTCondor, Parsl, Slurm, triple Slurm, etc.); HTCondor is default.
+  wmsServiceClass: lsst.ctrl.bps.htcondor.HTCondorService
+  
+  # Specify compute site and specific site settings.
+  computeSite: s3df
   site:
-    ncsapool:
+    s3df:
       profile:
         condor:
-          +Walltime: 28800
-  # Memory allocated for each quantum, in MBs
+          +Walltime: 7200
+          
+  # Memory allocated for each quantum, in MBs; 2048 is default.
```

Comment:
Not sure if the comma is needed but I do see that it is that way in the prior example.